src/static/swagger.json

```diff
@@ -713,25 +828,57 @@
                 }
             }
         },
-        "/account": {
+        "/presentation/verify": {
             "post": {
                 "tags": [
-                    "Account"
+                    "Presentation"
                 ],
-                "summary": "Create a client",
-                "description": "<p>This endpoint verifies the JWT token and creates a customer if they don't exist</p>",
+                "summary": "Verify a credential presentation",
+                "description": "<p>This endpoint verifies the credential presentation. As input it takes the entire presentation itself</p>",
+                "operationId": "presentation",
                 "security": [
                     {
                         "bearerAuth": []
                     }
                 ],
+                "requestBody": {
+                    "content": {
+                        "application/x-www-form-urlencoded": {
+                            "schema": {
+                                "$ref": "#/components/schemas/PresentationRequest"
+                            }
+                        },
+                        "application/json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/PresentationRequest"
+                            }
+                        }
+                    }
+                },
                 "responses": {
                     "200": {
                         "description": "The request was successful",
                         "content": {
                             "application/json": {
                                 "schema": {
-                                    "$ref": "#/components/schemas/Customer"
+                                    "$ref": "#/components/schemas/IVerifyResult"
+                                },
+                                "example": {
+                                    "verified": true,
+                                    "policies": {
+                                        "aud": false,
+                                        "audience": false
```

Comment:
What's the difference between `aud` and `audience`?

Reply:
There was an error in swagger, which overrode the example fixed it. Thanks